### PR TITLE
US003 - View Capability a role belongs to

### PR DESCRIPTION
**Pull request by OdhranHague**
As a Kainos employee

I want to be able to see what capability each role falls within
So that I can see what capability the role is within Kainos corporate structure

Acceptance Criteria

Update the job roles list page to show the capability name each role belongs to

- Updated getRoles query to include the "CapabilityName" field
- Updated "Role" class constructor to include capability name

### CODE REVIEW

**Comment by jkelso662** [comment]
Would be good to always have a screenshot for UI PRs :)

File: views/roles.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for role in roles %}
+                    <tr>
+                        <td>{{role.roleName}}</td>
+                        <td>{{role.specification}}</td>
+                        <td>{{role.capabilityName}}</td>
+                        <td>{{role.responsibilities}}</td>
+                        <td><a href="role.sharepointLink">Sharepoint</a></td>
```
Comment (jkelso662) [inline views/roles.html:40]: 🐛

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Think I spotted a bug might need checked out. Also you have a duplicated Role.ts file. Sort these and maybe and a screenshot of your UI and can get this approved :)

**Comment by OdhranHague** [comment]
- UI for viewing capability
Screenshot 2023-12-12 at 14 03 23

I have been running into issues with that duplicated role.ts file. The "Role.ts" is the only one there locally and when i pull down "role.ts", it never appears. I have just manually deleted the file and committed it through Github

**Review by Joseph-Magee — APPROVED** [review]
looks good

**Review by lullabelle — APPROVED** [review]
changes approved

**Review by jkelso662 — APPROVED** [review]

**Review by UserConor — APPROVED** [review]
Looks good, but will need the new UI implemented once it's complete

**Pull request merged**